iso8583.gemspec

```diff
@@ -1,29 +1,26 @@
-# -*- mode: ruby; encoding: utf-8; tab-width: 2; indent-tabs-mode: nil -*-
 lib = File.expand_path('../lib/', __FILE__)
 $:.unshift lib unless $:.include?(lib)
 
 require "iso8583/version"
  
 Gem::Specification.new do |s|
-  s.name        = "iso8583"
-  s.version     = ISO8583::VERSION
-  s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Tim Becker", "Slava Kravchenko"]
-  s.email       = ["tim.becker@kuriositaet.de","cordawyn@gmail.com"]
-  s.homepage    = "http://github.com/a2800276/8583/"
-  s.summary     = "Ruby implementation of ISO 8583 financial messaging"
-  s.description = "Ruby implementation of ISO 8583 financial messaging"
+	s.name         = "iso8583"
+	s.version      = ISO8583::VERSION
+	s.platform     = Gem::Platform::RUBY
+	s.authors      = ['Tim Becker', 'Slava Kravchenko', 'Manuel A. Güílamo']
```

Comment:
I've cleaned that bit up in the referenced branch :
      https://github.com/a2800276/8583/tree/MaG21-rspec-tests

and changed it to: 'see AUTHORS', I think we should just maintain a list of all contributors (not differentiating for now) there.